packages/page-staking/src/Actions/Account/ListNominees.tsx

```diff
@@ -53,7 +62,10 @@ function ListNominees ({ nominating, stashId }: Props): React.ReactElement<Props
         </Expander>
       )}
       {nomsWaiting && nomsWaiting.length !== 0 && (
-        <Expander summary={t<string>('Waiting nominations ({{count}})', { replace: { count: nomsWaiting.length } })}>
+        <Expander
+          help={t<string>('The validators that are not in the validator set. More nominations are required for them to get in.')}
```

Comment:
These are waiting validators or validators that have chilled.

Reply:
```suggestion
          help={t<string>('The validators that are not in the validator set because they need more nominations or because they have willingly stop validating.')}
```